src/main/java/appeng/client/gui/AEBaseScreen.java

```diff
@@ -401,6 +382,21 @@ private void drawOptionalSlotBackground(MatrixStack matrixStack, IOptionalSlot s
         }
     }
 
+    // Convert global mouse x,y to relative Point
+    private Point getMousePoint(double x, double y) {
+        return new Point((int) Math.round(x - guiLeft), (int) Math.round(y - guiTop));
+    }
+
+    @Override
+    public boolean mouseScrolled(double x, double y, double wheelDelta) {
+        if (wheelDelta != 0) {
+            if (widgets.onMouseWheel(getMousePoint(x, y), wheelDelta)) {
```

Comment:
Is `onMouseWheel` being used in vanilla? If so that is fine, however I would prefer using the same name whe forwarding a call, if possible. Keeps it consistent. And if `mouseScrolled` is not vanilla, but we cannot change it now, probably add a `@TODO`

Reply:
We have to make a pass once we migrate to actual official names. MCP is all over the place.